proposals/lip-0057.md

```diff
@@ -150,6 +150,7 @@ We also define `ROUND_LENGTH` to be the length of a round, i.e.,  `ROUND_LENGTH
 
 | Name | Type | Validation | Description |
 |------|------|------------|-------------|
+| `BlockHeader` | object   | Must follow the [`blockHeaderSchema`][lip-0055#block-header-json-schema] schema defined in LIP 0055.| The header of a block. |
```

Comment:
Note that the LIP already uses a type `Header`, e.g., in `verifyBlockSignature(header: Header)`. It would be nice to use one consistent type.

Reply:
Good catch, fixed.